src/home_robot/home_robot/perception/encoders/clip_encoder.py

```diff
@@ -9,8 +9,10 @@
 import torch
 from PIL import Image
 
+from .base_encoder import BaseImageTextEncoder
 
-class ClipEncoder:
+
+class ClipEncoder(BaseImageTextEncoder):
```

Comment:
good change - can we make sure places we were asking for ClipEncoder is switched to BaseImageTextEncoder?